pkg/modules/observability.go

```diff
@@ -11,18 +11,20 @@ package modules
 import (
 	"context"
 	"fmt"
+
 	"strconv"
 	"strings"
 
+	corev1 "k8s.io/api/core/v1"
```

Comment:
Why are we rearranging some of the k8s.io imports but leaving some of them down there

Reply:
This change occurred because of a merge conflict.